common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -249,6 +249,12 @@ def student_view(self, context):
                 return Fragment(special_html)
         return self._student_view(context, banner_text)
 
+    def anonymous_view(self, context):
+        """
+        Renders the anonymous view of the block in the LMS.
+        """
+        return self._student_view(context or {}, None, ANONYMOUS_VIEW)
```

Comment:
Instead of `context or {}`, set the default `def anonymous_view(self, context={}):`.

Reply:
@symbolist I don't think it's ever a good idea to have a mutable value as a default on a function or method.

Reply:
@xitij2000 You are absolutely right on that. Not sure what I had in mind when I wrote that. No change needed here.

Reply:
👍